### PR TITLE
L2D20A2

### PR DESCRIPTION
**Pull request by evyryan**
fixed fatheads
added question headers
some formatting changes

### CODE REVIEW

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 1]
```diff
@@ -176,14 +177,15 @@
             </div>
             <div xml:id="kzz7yh-f58876-Dd1e324">
                <head xml:id="kzz7yh-f58876-Hd1e326">Quaestio 2</head>
+               <head type="question-title">Utrum pueri statem cum nati fuissent habuissent perfectum membrorum virsum</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:180]:
```suggestion
               <head type="question-title">Utrum pueri statim cum nati fuissent habuissent perfectum membrorum usum</head>
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 2]
```diff
@@ -176,14 +177,15 @@
...
                <p xml:id="kzz7yh-f58876-d1e328">
-      Quaesio II 
-      <lb ed="#V" n="98"/>Ecundo queritur vtrum pueri statem cum 
-      nati<lb ed="#V" n="99" break="no"/>fuissent habuissent perfectum 
-      <lb ed="#V" n="100"/>membrorum vsum: et videtur quod sic. quaedam animalia statim 
-      <lb ed="#V" n="101"/>cum nascunt habent sufficientem membrorum vsum: ergo cum 
-      homo<lb ed="#V" n="102" break="no"/>sit perfectior ceteris animalibus si peccatum non fuisset statim cum 
-      nasce<lb ed="#V" n="103" break="no"/>retur non tantum sufficientem: sed persfactum membrorum vsum haberet.
+      Quaestio II <!--Word "vsum" I am confused about, occurs several times in this paragraph-->
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:182]:
```suggestion
      Quaestio II 
```

Comment (RobbieBuhite) [reply kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:182]: The word is "usum," as in "use" as a noun.

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 3]
```diff
@@ -176,14 +177,15 @@
...
+      <lb ed="#V" n="98"/>SEcundo queritur vtrum pueri statem cum nati
+      <lb ed="#V" n="99"/>fuissent habuissent perfectum 
+      <lb ed="#V" n="100"/>membrorum virsum: et videtur quod sic. quaedam animalia statim
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:185]:
```suggestion
      <lb ed="#V" n="100"/>membrorum vsum: et videtur quod sic. quaedam animalia statim 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 4]
```diff
@@ -176,14 +177,15 @@
...
-      nati<lb ed="#V" n="99" break="no"/>fuissent habuissent perfectum 
-      <lb ed="#V" n="100"/>membrorum vsum: et videtur quod sic. quaedam animalia statim 
-      <lb ed="#V" n="101"/>cum nascunt habent sufficientem membrorum vsum: ergo cum 
-      homo<lb ed="#V" n="102" break="no"/>sit perfectior ceteris animalibus si peccatum non fuisset statim cum 
-      nasce<lb ed="#V" n="103" break="no"/>retur non tantum sufficientem: sed persfactum membrorum vsum haberet.
+      Quaestio II <!--Word "vsum" I am confused about, occurs several times in this paragraph-->
+      <lb ed="#V" n="98"/>SEcundo queritur vtrum pueri statem cum nati
+      <lb ed="#V" n="99"/>fuissent habuissent perfectum 
+      <lb ed="#V" n="100"/>membrorum virsum: et videtur quod sic. quaedam animalia statim 
+      <lb ed="#V" n="101"/>cum nascunt habent sufficientem membrorum virsum: ergo cum homo
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:186]:
```suggestion
      <lb ed="#V" n="101"/>cum nascunt habent sufficientem membrorum vsum: ergo cum homo
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 5]
```diff
@@ -176,14 +177,15 @@
...
+      <lb ed="#V" n="102"/>sit perfectior ceteris animalibus si peccatum non fuisset statim cum 
+      nasce<lb ed="#V" n="103" break="no"/>retur non tantum sufficientem: sed persfactum membrorum virsum haberet.
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:188]:
```suggestion
      nasce<lb ed="#V" n="103" break="no"/>retur non tantum sufficientem: sed persfactum membrorum vsum haberet.
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 6]
```diff
@@ -201,9 +203,9 @@
       tal<lb ed="#V" n="112" break="no"/>infirmitas videtur ex vitio prouenisse: ergo non humissent infirmitatem carnis. 
    </p>
                <p xml:id="kzz7yh-f58876-d1e368">
-                  <lb ed="#V" n="113"/>Contra perfcio virium corporalium motiuarum dependet a 
-      per<lb ed="#V" n="114" break="no"/>fectione organorum. Sed in statu innoce filii non fuist 
-      <lb ed="#V" n="115"/>sent nati cum perfacta quantitate organorum: quia nati fuissent peruli: ergo 
+                  <lb ed="#V" n="113"/>Contra perficio virium corporalium motiuarum dependet a 
+      per<lb ed="#V" n="114" break="no"/>fectione organorum. Sed in statu innocente filii non
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:207]:
```suggestion
      per<lb ed="#V" n="114" break="no"/>fectione organorum. Sed in statu innocentie filii non 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 7]
```diff
@@ -278,9 +281,9 @@
       iusti<lb ed="#V" n="25" break="no"/>tiam originalem a sua natiuitate habuissent. 
    </p>
                <p xml:id="kzz7yh-f58876-d1e508">
-                  <lb ed="#V" n="26"/>Contra Hug. libro e de sacra. parte. 6 ca. 33. loquens de 
+                  <lb ed="#V" n="26"/>Contra <ref>Hug. libro i. de sacra. parte. 6. ca. 23.</ref> loquens de 
       <lb ed="#V" n="27"/>paruulis praedictis dicit quod qui sine peccato 
-      ge<lb ed="#V" n="28" break="no"/>nerarentur: conseqens est omnino quod est sine peccato nascerentur: ergo 
+      ge<lb ed="#V" n="28" break="no"/>nerarentur: conseqens est oino<!--word--> quod est sine peccato nascerentur: ergo
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:286]:
```suggestion
      ge<lb ed="#V" n="28" break="no"/>nerarentur: consequens est omnino quod est sine peccato nascerentur: ergo 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 8]
```diff
@@ -385,9 +389,10 @@
       <lb ed="#V" n="98"/>a principio habuissent perfectum rationis vsum. Sed perfectio 
       <lb ed="#V" n="99"/>scientie non est sine perfectione vsus rationis: ergo non fuissent 
       <lb ed="#V" n="100"/>nati in scientia perfecti. 
-      <lb ed="#V" n="101"/>Ad hanc quaestionem dicunt quidam: quod paruuli 
+      <!--TODO add paragraph-->
+      <lb ed="#V" n="101"/>Ad hanc quaestionem dicunt quidam: quod paruuli non 
       fuis<lb ed="#V" n="102" break="no"/>sent nati cum perfectione scientie nec quantum 
-      <lb ed="#V" n="103"/>ad vsum nec quantum ad habitum: sed in processu temporis 
+      <lb ed="#V" n="103"/>ad virsum nec quantum ad habitum: sed in processu temporis
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:395]:
```suggestion
      <lb ed="#V" n="103"/>ad vsum nec quantum ad habitum: sed in processu temporis 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 9]
```diff
@@ -414,9 +419,9 @@
       <lb ed="#V" n="124"/>remouetur et commotiones sedantur. 
     </p>
                <p xml:id="kzz7yh-f58876-d1e764">
-                  <lb ed="#V" n="125"/>Ad primum in opositum cum dicitur: quod ignorantia peciea est etc. 
+                  <lb ed="#V" n="125"/>Ad primum in opositum cum dicitur: quod ignorantia peciea<!--word--> est etc.
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:422]:
```suggestion
                  <lb ed="#V" n="125"/>Ad primum in opositum cum dicitur: quod ignorantia pena est etc. 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 10]
```diff
@@ -414,9 +419,9 @@
       <lb ed="#V" n="124"/>remouetur et commotiones sedantur. 
     </p>
                <p xml:id="kzz7yh-f58876-d1e764">
-                  <lb ed="#V" n="125"/>Ad primum in opositum cum dicitur: quod ignorantia peciea est etc. 
+                  <lb ed="#V" n="125"/>Ad primum in opositum cum dicitur: quod ignorantia peciea<!--word--> est etc. 
       <lb ed="#V" n="126"/>dico quod verum est proprie loquendo de 
-      ignran<lb ed="#V" n="127" break="no"/>tia: sed secundum hug id est li. de sacr. per. 6 c. 26 non omnis qui aliquid nescit aut 
+      ingran<lb ed="#V" n="127" break="no"/>tia: sed secundum hug id est li. de sacr. per. 6 c. 26 non omnis qui aliquid nescit aut
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:424]:
```suggestion
      ingran<lb ed="#V" n="127" break="no"/>tia: sed secundum <ref>hug id est li. de sacr. per. 6 c. 26</ref> non omnis qui aliquid nescit aut 
```

File: kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml [hunk 11]
```diff
@@ -449,8 +454,8 @@
       <lb ed="#V" n="10"/>lium: etiam quantum ad habitum: ideo sunt soluenda. 
     </p>
                <p xml:id="kzz7yh-f58876-d1e829">
-                  <lb ed="#V" n="11"/>Ad primum dicendum quod illa aucoritas philosophi ntelligenda 
-      <lb ed="#V" n="12"/>est pro statu nature lapse et de lege communi. 
+                  <lb ed="#V" n="11"/>Ad primum dicendum quod illa aucoritas philosophi intelligenda 
+      <lb ed="#V" n="12"/>est pro statu naturae lapse et de lege communi.
```
Comment (RobbieBuhite) [inline kzz7yh-f58876/cod-ve09v2_kzz7yh-f58876.xml:458]:
```suggestion
      <lb ed="#V" n="12"/>est pro statu nature lapse et de lege communi. 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**